cmd/server/config.go

```diff
@@ -11,9 +10,12 @@ import (
 )
 
 const (
-	ocidRoot            = "/var/lib/ocid"
+	ocidRoot            = "/var/lib/containers"
+	ocidRunRoot         = "/var/run/containers"
 	conmonPath          = "/usr/libexec/ocid/conmon"
-	pausePath           = "/usr/libexec/ocid/pause"
+	pauseImage          = "kubernetes/pause"
```

Comment:
Yeah, so my main concern is that I wanted to make sure that it is possible to use `ocid` **without** needing to pull images from anywhere. This is something that will be a requirement from SUSE's side eventually (so we can audit all of the base images).